webapp/views.py

```diff
@@ -253,7 +253,7 @@ def bulk_upsert_cve(*args, **kwargs):
 
     for data in cves_data:
         update_cve = False
-        cve = db_session.query(CVE).filter(CVE.id == data["id"]).one_or_none()
+        cve = db_session.query(CVE).get(data["id"].upper())
```

Comment:
Note: Because testing this function requires authorization, this line was not tested before pushing this commit

Reply:
Simliar to my other comment, try changing this function

https://github.com/mtruj013/ubuntu-com-security-api/blob/fix-case-sensitivity/tests/test_routes.py#L161-L179

To add some lowercase IDs, and then run the python tests.